packages/runtime/lib/app.js

```diff
@@ -27,6 +27,13 @@ class PlatformaticApp {
     this.#logger = logger
   }
 
+  getState () {
+    return {
+      started: this.#started,
```

Comment:
This is basically exposing an implementation detail of the runtime. I'm not sure if you need both flags. You could create an enum of stopped, started, restarting. If you need more states in the future, they could be added.